crates/turbopack-ecmascript-runtime/js/src/build/runtime.ts

```diff
@@ -62,10 +70,24 @@ function loadChunk(chunkPath: ChunkPath) {
   }
 }
 
-function loadChunkAsync(source: SourceInfo, chunkPath: string): Promise<void> {
+async function loadChunkAsync(
+  source: SourceInfo,
+  chunkData: ChunkData
+): Promise<any> {
+  if (typeof chunkData === "string") {
+    return loadChunkPathAsync(source, chunkData);
+  } else {
+    return loadChunkPathAsync(source, chunkData.path);
+  }
+}
+
+function loadChunkPathAsync(
+  source: SourceInfo,
+  chunkPath: string
```

Comment:
Nit: this could just be a `ChunkData`, and then we eliminate the duplicate if condition in `loadChunk`/`loadChunkAsync`.